docs/content/02-tutorials/07-odcr.md

```diff
@@ -0,0 +1,94 @@
++++
+title = "d. On-Demand Capacity Reservation 🤖"
+weight = 28
++++
+
+On-Demand Capacity Reservations (ODCR's) are used to reserve capacity, this is particularly useful for instances with constrained capacity such as `p4dn.24xlarge`. This tutorial assumes you're familiar with the [steps here](https://docs.aws.amazon.com/parallelcluster/latest/ug/launch-instances-odcr-v3.html).
+
+## Setup
+
+To setup a ODCR with AWS ParallelCluster, we'll use the managed post-install script. This creates a file `/opt/slurm/etc/pcluster/run_instances_overrides.json` on the head node with the queue, compute resource, and capacity reservation group arn filled in.
+
+1. Create a new cluster and on the **HeadNode** configuration screen, click on the "Advanced" dropdown and add in the `On-Demand Capacity Reservation` script:
+
+    ![On-Demand Capacity Reservation](odcr/post-install.png)
+
+    Fill in the parameters accordingly:
+
+    | **Parameter**                   | **Description**                                                                                                                                                         |
+    |---------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+    | Queue Name                      | Name of the queue you want to use the capacity reservation with.                                                                                                        |
+    | Compute Resource                | Name of the compute resource within the queue, this is typically `[queue_name]-[instance_type]` without the `.` in the instance type. i.e. `queue0-p4d24xlarge`         |
+    | Capacity Reservation Group Name | This is the name of the Capacity Group you created in [Step 1](https://docs.aws.amazon.com/parallelcluster/latest/ug/launch-instances-odcr-v3.html#odcr-create-cluster) |
```

Comment:
It is also possible to do ODCR with a reservation ID so we should consider parameterizing that

Reply:
ah yes, I think that's actually better since it's simpler for the customer to just put in the reservation id

Reply:
I was thinking that we would support both -- by specifying the type (and default it to `CapacityReservationResourceGroupArn`) so that it can be changed. Both are needed depending on the type of reservation.